labs/session2-codes/hadoop-examples/Difference.java

```diff
@@ -0,0 +1,77 @@
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.io.Text;
+import org.apache.hadoop.mapreduce.Job;
+import org.apache.hadoop.mapreduce.Mapper;
+import org.apache.hadoop.mapreduce.Reducer;
+import org.apache.hadoop.mapreduce.lib.input.FileInputFormat;
+import org.apache.hadoop.mapreduce.lib.input.FileSplit;
+import org.apache.hadoop.mapreduce.lib.output.FileOutputFormat;
+
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Created by TeamZero on 23/01/20.
+ * input: inputs/ralational-algebra-op-input/
+ * output: hadoop-output/ralational-algebra-op-out/difference/
+ * local -> file:/home/mojtaba/Desktop/hadoop-examples/inputs/ralational-algebra-op-input/
+ * hadoop -> hdfs://namenode:port/[file address]
+ * use same pattern for output
+ */
+public class Difference {
+
+    public static class DifferenceMapper extends Mapper<Object , Text, Text, Text>{
```

Comment:
Some comments would be welcome, e.g., to explain that the difference is here computed on the year attribute only.